cmd/query/app/http_handler.go

```diff
@@ -115,6 +127,10 @@ func (aH *APIHandler) RegisterRoutes(router *mux.Router) {
 	// TODO - remove this when UI catches up
 	aH.handleFunc(router, aH.getOperationsLegacy, "/services/{%s}/operations", serviceParam).Methods(http.MethodGet)
 	aH.handleFunc(router, aH.dependencies, "/dependencies").Methods(http.MethodGet)
+	aH.handleFunc(router, aH.latencies, "/metrics/latencies/{%s}", servicesParam).Methods(http.MethodGet)
```

Comment:
Are you particularly set on passing service in the URL?  We tried to move away from that (see L126)

Reply:
Nope, I'm not particularly attached to passing service as a URL path segment. Just thought that having services in the URL path is an affordance to suggest it is a mandatory parameter.

What was the reason behind moving away from it? I only found this comment, but couldn't find the discussion on why there was final agreement on moving in this direction: https://github.com/jaegertracing/jaeger/pull/52#discussion_r103296286

Reply:
> What was the reason behind moving away from it?

1. part of my holy war on high cardinality REST URLs (when params are encoded in the URL we can't use URL as the span name)
2. when the service name is part of the URL people sometimes tend to think it can go there unencoded, which breaks when the service name contains a slash

in the end though - not very strong reasons